fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -432,6 +432,9 @@ private RollupJobV2 createMaterializedViewJob(String mvName, String baseIndexNam
         List<Column> newMVColumns = Lists.newArrayList();
         int numOfKeys = 0;
         if (olapTable.getKeysType().isAggregationFamily()) {
+            if (addMVClause.getMVKeysType() != KeysType.AGG_KEYS) {
```

Comment:
Do we allowed to create mv on unique key table?
if yes, how to create it?

Reply:
We allowed to create mv on unique key if and only if materialized view of full key column.

Reply:
the unique key table has unique feature , we need mv to aggregate  such as  bitmap_union(to_bitmap(user_id)),but  it's wrong 。
example:

CREATE TABLE t
(
    event_day DATE,
    momo_id INT ,
    event_name VARCHAR(100),
    access_timestamp BIGINT
)
UNIQUE KEY(event_day,momo_id, event_name,access_timestamp)
PARTITION BY RANGE(event_day) ()
DISTRIBUTED BY HASH(user_id)
PROPERTIES
(
    "dynamic_partition.enable" = "true",
    "dynamic_partition.time_unit" = "DAY",
    "dynamic_partition.end" = "3",
    "dynamic_partition.prefix" = "p",
    "dynamic_partition.buckets" = "8",
    "storage_type"="column"
);

create materialized view mv as
select  event_day,bitmap_union(to_bitmap(user_id))  from t group by event_day ;